types/analytics-node/analytics-node-tests.ts

```diff
@@ -40,11 +40,9 @@ function testIdentify(): void {
       plan: 'Enterprise',
       friends: 42
     }
-  }, (err, data) => {
```

Comment:
It does pass `data` to the callback https://github.com/segmentio/analytics-node/blob/a78b2ac52a58a389e0880890b0e972290dab3a1a/index.js#L256

Reply:
@thomasthiebaud that's the flush callback, not the message callback. The flush signature is untouched.

Feel free to actually test this. Only an error argument is passed for messages.